docs/developer-docs/smart-contracts/advanced-features/randomness.mdx

```diff
@@ -61,25 +61,13 @@ let (randomBytes,): (Vec<u8>,) = ic_cdk::api::call(Principal.management_canister
 <AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
 
 ```typescript
-import { IDL, postUpgrade, query, update } from 'azle';
-
-let redeployed = false;
+import { IDL, query, update } from 'azle';
```

Comment:
Needs to look more like the previous comment:

```typescript
import { IDL, setTimer, update } from 'azle';

export default class {
    @update([], IDL.Nat64)
    createTimer(): bigint {
        const timerId = setTimer(1_000n, () =>
            console.log('timer callback called')
        );

        return timerId;
    }
}
```

Reply:
Revised example